protocol/contracts/v2/TempleDebtToken.sol

```diff
@@ -66,19 +66,19 @@ contract TempleDebtToken is ITempleDebtToken, TempleElevatedAccess {
     uint96 public override baseRate;
```

Comment:
The change to dToken was driven by a gas optimisation and simpler code. It now follows the pattern which:
  1/ fill a memory cache upfront, updating any state (like compounded interest) at that point
  2/ passing that memory through functions as required
  3/ only updating state again as required at the end

Which simplifies function interfaces and found some gas optimisations where it was previously sload/sstore more than it had to.